airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -269,12 +266,8 @@ private static WorkspaceRead buildWorkspaceRead(final StandardWorkspace workspac
         .notifications(NotificationConverter.toApiList(workspace.getNotifications()))
         .defaultGeography(Enums.convertTo(workspace.getDefaultGeography(), Geography.class));
     // Add read-only webhook configs.
-    final Optional<WebhookOperationConfigs> persistedConfigs = Jsons.tryObject(
```

Comment:
to make sure I'm following, we are able to remove the `Jsons.tryObject` call because we handle this in the `WorkspaceWebhookConfigsConverter.java` class now?

Reply:
Yep - basically `tryObject` was always failing (returning `Optional.empty()`) because the JsonNode _doesn't_ actually fit the schema.